api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -47,22 +48,39 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
-        String locationUserPropertyName = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME);
-        if(StringUtils.isBlank(locationUserPropertyName)) {
-            Context.getAdministrationService().setGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
-                    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
-            if(log.isDebugEnabled()) {
-                log.debug("RefApp Location global property created with value - " + LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
-            }
-        }
-        else if(StringUtils.isNotBlank(locationUserPropertyName) && !locationUserPropertyName.equals(LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME)) {
-            log.warn("RefApp Location global property already exist in the system with different value(" + locationUserPropertyName +
-                    "). Exiting from creating new global property with the value " + LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
-        }
-        log.info("Location Based Access Control Module started");
+	    PersonAttributeType personAttributeType = getPersonAttributeType();
+	    Context.getPersonService().savePersonAttributeType(personAttributeType);
+	    addGlobalProperty(LocationBasedAccessConstants.LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME,
+			    LocationBasedAccessConstants.PERSONATTRIBUTETYPE_UUID);
+	    addGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
+			    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
+	    log.info("Location Based Access Control Module started");
     }
 
-    /**
+	private PersonAttributeType getPersonAttributeType() {
```

Comment:
As this is only responsible for creating personAttributeType for LBAC, can you reflect that in the method name, and move to utils?